tests/integration/test_upgrade.py

```diff
@@ -95,6 +106,7 @@ async def deploy_old_version_fixture(
     assert ops_test.model
 
     async def deploy_wordpress():
+        """Helper function to deploy wordpress charm."""
```

Comment:
Maybe `Deploy wordpress charm.`? Similar for the other docstrings, does noting it is a helper function add value to the docstring?